### ISSUE TITLE
PHP 8.1 support

### ISSUE THREAD

**Issue by Venorcis** [Ocramius/ProxyManager#719]
"ocramius/proxy-manager 2.13.0 requires php ~7.4.1 || ~8.0.0 -> your php version (8.1.0) does not satisfy that requirement."

**Comment by Ocramius** [comment]
Duplicate of #716

**Comment by Venorcis** [comment]
> Duplicate of #716

that issue is already closed though, maybe good to keep it open so it's visible?

**Comment by Ocramius** [comment]
Not really: support work still needs to start, and I know I have to do it.

Mostly involves tons of added tests and edge case scenarios around `never` and similar, for which I still need to formulate the actual tasks.

The upstream `laminas/laminas-code` work needs to be completed first anyway.
